### PR TITLE
feat(STONEINTG-660): rename CLAIR_SCAN_RESULT to SCAN_OUTPUT

### PR DESCRIPTION
**Pull request by jsztuka**
# Before you complete this pull request ...

Look for any open pull requests in the repository with the title "e2e-tests update" and 
see if there are recent e2e-tests updates that will be applicable to your change.

Renaming of results for clair-scan required bumping up the version of task itself. Migration markdown created together with it to let customer know what to expect.

### CODE REVIEW

**Comment by jsztuka** [comment]
EC was not detecting new version of clair-scan with updated results. To fix this PR https://github.com/enterprise-contract/ec-policies/pull/1133 was created. Previousl caused problem should be solved.

File: policies/all-tasks.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ sources:
       required_task_results:
         # Certain EC rules rely on the presence of this result when validating an image.
         - task: clair-scan
-          result: CLAIR_SCAN_RESULT
+          result: SCAN_OUTPUT
```
Comment (zregvart) [inline policies/all-tasks.yaml:18]: With the new policy, we want to configure that for 0.1 the result name is `CLAIR_SCAN_OUTPUT`, and for all other versions it should be `SCAN_OUTPUT`, something like this:

```suggestion
          result: CLAIR_SCAN_OUTPUT
          version: 0.1
        - task: clair-scan
          result: SCAN_OUTPUT
```

**Comment by jsztuka** [comment]
Created https://github.com/konflux-ci/e2e-tests/pull/1386 in order to fix the e2e tests.

**Review by dirgim — APPROVED** [review]
LGTM

**Review by jencull — APPROVED** [review]
LGTM

File: hack/build-and-push.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 set -e -o pipefail
-
+set -x
```
Comment (zregvart) [inline hack/build-and-push.sh:4]: Debugging or left on purpose?

**Review by zregvart — APPROVED** [review]

**Review by dheerajodha — APPROVED** [review]

File: hack/build-and-push.sh (hunk 2 above)
Comment (jsztuka) [reply hack/build-and-push.sh:4]: Debugging, removing this one, thanks!

**Review by kasemAlem — APPROVED** [review]

**Comment by zregvart** [comment]
FYI, [the upgrade to 0.2](https://github.com/enterprise-contract/ec-cli/pull/1959) in EC went without issues

**Pull request merged**